apps/docs/app/globals.css

```diff
@@ -1,5 +1,6 @@
 @import url("./tokens.css");
 @import url("./rehype.css");
+@import url("@hopper-ui/components/index.css");
```

Comment:
where was this before? just wondering why we had to add it here.

Reply:
This is a mistake that should not have made it in the PR.